app/src/main/kotlin/com/softteco/template/ui/feature/settings/SettingsScreen.kt

```diff
@@ -100,6 +101,19 @@ private fun ScreenContent(
                 },
             )
             Divider()
+            ListItem(
+                modifier = Modifier.clickable {
+                    context.sendMail(recipient = "softteco.os.dev@gmail.com", subject = "User Inquiry or Feedback")
+                },
+                headlineContent = { Text(text = stringResource(id = R.string.contact_us)) },
+                trailingContent = {
+                    Icon(
+                        Icons.Sharp.ArrowForwardIos,
+                        contentDescription = "Settings description"
```

Comment:
We translated a contentDescription in this app before. Let's use string res for this as well. Also, it should be different names for contentDescription for different list items.